src/collections/blog/2022/2022-20-06-managing-multiple-clusters-with-meshery/index.mdx

```diff
@@ -0,0 +1,99 @@
+ - Kubernetes
+type: Blog
+published: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import Switcher from "./context-switcher.png";
+import Deploy from "./deploy-modal.png";
+import Cluster2 from "./meshmap-cluster2.png";
+import Cluster1 from "./meshmap-cluster1.png";
+import Settings from "./settings.png";
+import Cluster from "./cluster-mgmt.png";
+import Flush from "./flush-meshsync.png";
+import CoreArch from "./meshery-core-architecture.png";
+import { Link } from "gatsby";
+
+<BlogWrapper>
+
+<div class="intro">
+<p>
+From multi-mesh to now multi-cluster, <Link to="/meshery">Meshery</Link> is continuously expanding its capability to give developers, operators, and security engineers more control over their infrastructure. In this post, we'll take a look behind the scenes at how each component in Meshery's architecture plays a role in the management of many Kubernetes clusters.
+</p>
+</div>
+
+## Philosophy behind Meshery's multi-cluster management approach
+
+While designing Meshery for the world of many service meshes, and many Kubernetes clusters, much care has been taken to ensure that Meshery is an <a href="https://docs.meshery.io/extensibility">extensible management platform</a>, ready for handling new types of infrastructure and new use cases rapidly through its plugin model. Under the hood, <a href="https://docs.meshery.io">Meshery Server</a> acts as a delegator of operations by figuring out which Meshery Adapter registered its capability against the given operation. The operation is then sent to that given component (like one of Meshery’s service mesh adapters,e.g. Istio adapter) via a gRPC call. <img src={CoreArch} alt="deploy modal"  class="image-right" /> When the operation involves a kubernetes cluster(s), the kubeconfig(s) is sent as a parameter to the RPCs. It is then the job of the handling adapter to respect that and perform the operation across the passed clusters from kubeconfigs as needed. The operations not requiring a kubeconfig are managed through the same RPC, with the only difference being that the handling component would ignore the `kubeconfigs` field altogether making the system work not just for kubernetes, but for other cloud native use cases. This approach of reusing the same RPC for different types of requests is pretty common and sometimes debatable with the other approach of being strict with the RPCs. This is what makes Meshery completely pluggable and extensible.
+
+## Using multi cluster with Meshery
+From a client's perspective, there are two uses of the multi context feature in general. While deploying a <Link to="/meshmap">MeshMap</Link> design or performing any other operation on their cluster(s), selecting any number of kubernetes contexts will allow them to uniformly and parallely perform the operation across the clusters. And while visualizing the state of their cluster(s), the same context switcher will allow them to filter across the clusters whose view they want to see.
```

Comment:
Capitalize all instances of Kubernetes. 

Expand on what the Meshery Operator is.